opus/application/static_media/js/browse.js

```diff
@@ -38,15 +37,15 @@ var o_browse = {
     imageSize: 100,     // default
     maxCachedObservations: 1000,    // max number of obserations to store in cache; at some point, we can probably figure this out dynamically
 
-    limit: 100,  // results per page
+    // set default to 200 so loadData will fetch enough number of data for the first time in large screen
```

Comment:
We should get rid of this default limit entirely. There's no reason countGalleryImages() can't be called in the beginning to set the limit to the proper value.